app/services/phone_number_capabilities.rb

```diff
@@ -10,12 +10,25 @@ def self.load_config
   attr_reader :phone, :phone_confirmed
 
   def self.translated_international_codes
-    translated_international_codes_data = {}
-    INTERNATIONAL_CODES.each do |k, value|
-      translated_international_codes_data[k] =
-        value.merge('name' => I18n.t("countries.#{k.downcase}"))
+    if Rails.env.development?
+      translated_international_codes_data = {}
+      INTERNATIONAL_CODES.each do |k, value|
+        translated_international_codes_data[k] =
+          value.merge('name' => I18n.t("countries.#{k.downcase}"))
+      end
+      translated_international_codes_data
+    else
+      return @translated_intl_codes_data[I18n.locale] if defined?(@translated_intl_codes_data)
+      @translated_intl_codes_data = {}
+      I18n.available_locales.each do |locale|
+        INTERNATIONAL_CODES.each do |k, value|
+          @translated_intl_codes_data[locale] ||= {}
+          @translated_intl_codes_data[locale][k] =
```

Comment:
small micro-optim, we could save line if we did this:
```suggestion
      @translated_intl_codes_data = Hash.new { |h, k| h[k] = {} }
      I18n.available_locales.each do |locale|
        INTERNATIONAL_CODES.each do |k, value|
          @translated_intl_codes_data[locale][k] =
```